students/kondel_tomasz/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,20 @@
+x_1 = int(input())
+y_1 = int(input())
+x_2 = int(input())
+y_2 = int(input())
+
+# there are 8 possible moves from point 1
+
+pos_moves = [x_1 + 1, y_1 + 2], [x_1 + 1, y_1 - 2], [x_1 - 1, y_1 + 2],\
```

Comment:
You don't modify that, so `tuple` would be more appropriate (parentheses instead of brackets)